waku/v2/peermanager/peer_manager.go

```diff
@@ -170,6 +170,10 @@ func (pm *PeerManager) connectToRelayPeers() {
 			numPeersToConnect = notConnectedPeers.Len() - 1
 		}
 
+		if numPeersToConnect <= 0 {
```

Comment:
Hi @harsh-98 : Thanks for reviewing this code, My initial attempt with this PR was to quickly patch and stop this panic because this was a release blocker for status-mobile.

I am exploring your solution and will share a follow up PR this week incorporating this feedback !
Thanks  !

Reply:
@siddarthkay , I am working on pushing changes as recommended by @harsh-98 . 
So, you don't need to worry.